# Intervalos de confiança 
## Intervalo de confiança para média com σ conhecido



Importando Bibliotecas

In [1]:
import scipy.stats as stats

**Exemplo 1**

Em uma indústria de cerveja, a quantidade de cerveja inserida em latas tem-se comportado como uma variável aleatória distribuída normalmente com média 350 ml e desvio padrão 3 ml. Após alguns problemas na linha de produção, suspeita-se que houve alteração na média. Uma amostra de 120 latas acusou média 346 ml. Encontre a estimativa pontual e construa um intervalo de confiança para o novo valor da quantidade média de cerveja inserida em latas, com nível de confiança de 95%, supondo que não tenha ocorrido alteração no desvio padrão do processo.

X: Quantidade média de cerveja inserida nas latas (em ml).

Dados disponíveis no enunciado:

-   X \~ N(350,3²)

-   n=120 latas → Tamanho da amostra

-   media=346ml → Quantidade média observada na amostra → Estimativa pontual!

-   σ=3ml → Desvio padrão da população de cervejas produzidas pela empresa.

-   1-α=0,95 → Nível de confiança


In [2]:
# Dados
media = 346
sigma = 3
alpha = 0.05
n = 120

In [7]:
# Valor crítico Z
zc = stats.norm.ppf(1 - alpha/2)
zc

1.959963984540054

In [4]:
round(n ** 0.5, 2)

10.95

In [9]:
# Erro amostral
erro = zc * sigma / (n ** 0.5)
erro

0.5367582431151471

Construir o intervalo para μ com 95% de confiança

In [10]:
# Intervalo de confiança
intervalo_confianca = (media - erro, media + erro)
print(intervalo_confianca)

(345.4632417568848, 346.5367582431152)


**Interpretação do intervalo de confiança:**

[345.46 ; 346.54] Podemos afirmar com 95% de confiança que o intervalo 345.46 ml a 346.54 ml contem o verdadeiro valor da quantidade média de cerveja inserida nas latas.


**Faça você mesmo:**

O diretor do comitê de admissão de uma universidade deseja estimar a média de idade de todos os estudantes aprovados no momento. Sabe-se que, de levantamentos anteriores, o desvio padrão da população é de 1,5 ano. Em uma amostra aleatória de 20 estudantes, a idade média encontrada foi de 22,9 anos.

X: Idade de todos os estudantes aprovados .

Dados disponíveis no enunciado:

- n=20 estudantes → Tamanho da amostra

- media= 22,9 anos→ Quantidade média observada na amostra

- σ= 1,5 ano → Desvio padrão da população.

a)  Com base nessa amostra, qual é a estimativa pontual da idade média dos estudantes aprovados?

b)  Construa um intervalo de 90% de confiança para a idade média da população.


## Intervalo de confiança para média com σ desconhecido


**Exemplo 1**

Com um teste destrutivo, as amostras são destruídas no processo de teste. O teste de colisão de carros é um exemplo muito dispendioso de teste destrutivo. Suponha que tenhamos feito teste de colisão em 12 carros esporte sob uma diversidade de condições que simulam colisões típicas. A análise dos 12 carros danificados resulta em custos de conserto com média igual a US\$26.227,00 e um desvio padrão de US\$15.873,00. Sabe-se que os dados têm distribuição aproximadamente normal.

X: Custo de conserto pós colisão .

Dados disponíveis no enunciado:

-   n=12 → Tamanho da amostra

-   media= 26.227,00 → Quantidade média observada na amostra

-   s= 15.873,00 → Desvio padrão AMOSTRAL.

Determine:

a)  A melhor estimativa pontual de μ, o custo médio de conserto de todos os carros esporte envolvidos em colisões.

    US\$26227,00

b)  A estimativa intervalar de 95% de confiança para μ

In [11]:
# Dados
media = 26227
s = 15873
n = 12

# Valor crítico t
tc = stats.t.ppf(1 - alpha/2, df=n-1)

# Erro amostral
erro = tc * s / (n ** 0.5)

# Intervalo de confiança
intervalo_confianca = (media - erro, media + erro)
print(intervalo_confianca)

(16141.776953040577, 36312.22304695942)


Considerando um nível de 95% de confiança podemos dizer que o intervalo de 16.141,78 a 36.312,22 contém o verdadeiro valor do custo médio para conserto de veículos submetidos a teste de colisão, considerando uma margem de erro de 10.085,22.

**Faça você mesmo:**

Em uma pesquisa de orçamento familiar desenvolvida pelo instituto ZX, solicitou-se a uma amostra de 16 domicílios de certa região que anotassem suas despesas com alimentação durante uma semana. O resultado foi uma despesa média de R\$330,00 com um desvio padrão de R\$40,00. Construa e interprete um intervalo de 98% de confiança para a verdadeira despesa média semanal com alimentação por domicílio de toda a região. Suponha que a população tenha uma distribuição aproximadamente normal.

X: Despesa média familiar .

Dados disponíveis no enunciado:

-   n=16 → Tamanho da amostra

-   media= 330,00 → Quantidade média observada na amostra

-   s= 40,00 → Desvio padrão AMOSTRAL.

## Intervalo de confiança para Proporção



**Exemplo 1**

Em uma linha de produção de certa peça cerâmica, colheu-se uma amostra de 125 itens, constatando-se 7 peças eram defeituosas. Com esses resultados amostrais, determine a estimativa intervalar de 87% de confiança da proporção de peças defeituosas produzidas pela empresa.

Dados disponíveis no enunciado:

-   n=125 itens → Tamanho da amostra

-   x=7→ Número de peças defeituosas observado na amostra

-   1-α=0,87 → Nível de confiança → 13%/2=6,5%=0,065

-   𝑝=x/n=7/125=0,056→ Proporção de peças defeituosas observada na amostra.

Suposições para o uso da distribuição normal:

-   n\. 𝑝\>5 → 125.0,056 =7 \>5 Ok!

-   n.(1- 𝑝) \>5 → 125.(1-0,056)=118 \> 5 Ok!

Interpretação do intervalo de confiança:


In [12]:
# Dados
x = 7
n = 125

# Valor crítico Z
zc = stats.norm.ppf(1 - (1 - 0.87)/2)

# Proporção
p = x / n

# Erro amostral
erro = zc * ((p * (1 - p)) / n) ** 0.5

# Intervalo de confiança
intervalo_confianca = (p - erro, p + erro)
print(intervalo_confianca)

(0.024862760653218824, 0.08713723934678118)


Considerando um nível de 87% de confiança podemos dizer que a proporção de peças defeituosas produzidas pela empresa pode variar de 2,48% a 8,71% , com uma margem de erro de 3,11%

***Refaça os cálculos utilizando 80% e 99% de confiança. Compare os resultados obtidos!***

**Faça você mesmo**

De uma amostra de 230 pessoas selecionadas aleatoriamente participantes de um grande evento, 145 disseram estar satisfeitas com o atendimento recebido. Os organizadores desse evento afirmar que a proporção de pessoas satisfeitas é igual a 70%. Com um nível de 92% de confiança, é possível contestar a afirmação dos organizadores?

Parâmetro: Proporção de pessoas satisfeitas com o atendimento recebido.

Dados disponíveis no enunciado:

-   n=230 pessoas → Tamanho da amostra

-   x=145 → Número de pessoas satisfeitas observado na amostra

-   1-α=0,92 → Nível de confiança

-   𝑝=x/n=145/230=0,06304→ Proporção de pessoas satisfeitas observada na amostra.

